config/production.json

```diff
@@ -54,5 +57,28 @@
   },
   "thegivingblock": {
     "apiUrl": "https://public-api.tgbwidget.com/v1"
+  },
+  "aws": {
+    "s3": {
+      "bucket": "opencollective-production"
+    }
+  },
+  "graphql": {
+    "cache": {
+      "enabled": true
+    }
+  },
+  "dbEncryption": {
+    "cipher": "AES"
+  },
+  "captcha": {
+    "enabled": true
+  },
+  "limits": {
+    "ordersPerHour": {
+      "perAccount": 100,
+      "perAccountForCollective": 10,
+      "perEmailForCollective": 5
```

Comment:
Same here; keeping that private would make sense to me. Unless we override in prod.